wpilibj/src/main/java/edu/wpi/first/wpilibj/Ultrasonic.java

```diff
@@ -248,7 +248,7 @@ public synchronized void close() {
    *                 is preferred, it can be implemented by pinging the sensors manually and waiting
    *                 for the results to come back.
    */
-  public void setAutomaticMode(boolean enabling) {
+  public static void setAutomaticMode(boolean enabling) {
```

Comment:
This change seems to be irrelevant to this PR?

Reply:
Correct, it would seem that two different PRs got squashed together to the same branch. This change is for PR #2419 , and issue #2417 . Is there a way to remove irrelevant commit from this PR?

Reply:
I reverted the irrelevant commit.

Reply:
You could just force-push it away.